docs/src/_includes/layouts/doc.html

```diff
@@ -101,7 +101,15 @@ <h1>{{ title }}</h1>
                 {% if edit_link %}
                     {{ edit_link }}
                 {% else %}
-                    {{ site.edit_link.start_with }}{{ page.inputPath }}
+                    {% if is_pre_release %}
+                        {% if PATH_PREFIX === '/docs/latest/' %}
```

Comment:
Can these two conditions ever be `true` at the same time? If the build is run for `/docs/latest/`, then the `version` field in `package.json` won't be a prerelease version?